ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/OpenApiStatisticsController.java

```diff
@@ -0,0 +1,46 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.admin.rest.controllers;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+@SuppressWarnings("java:S2479") // whitespace needed for examples
+@Tag(name = "Statistics")
+public interface OpenApiStatisticsController {
+
+	@Operation(summary = "Retrieve json with statistics of the LDES server")
+	@ApiResponse(responseCode = "200", content = {
+			@Content(mediaType = "application/json", schema = @Schema(implementation = String.class), examples = @ExampleObject(value = """
+					{
+					    "ingested member count": 4018,
+					    "current member count": 4018,
+					    "LDESes": [
+					        {
+					            "name": "mobility-hindrances",
+					            "ingested member count": 4018,
+					            "current member count": 4018,
```

Comment:
Frankly, to an outsider, this might be quite confusing..
The difference between Ingested members and current member count is only something for internal purposes, no?

Reply:
True, ingested might be beter as historical member count? or something like that